extensions/funqy/funqy-knative-events/runtime/src/main/java/io/quarkus/funqy/runtime/bindings/knative/events/JsonCloudEventImpl.java

```diff
@@ -62,8 +62,10 @@ public String id() {
     public String specVersion() {
         if (specVersion == null) {
             JsonNode specVersion = event.get("specversion");
-            if (specVersion != null) {
+            if (specVersion != null && isKnownSpecVersion(specVersion.asText())) {
                 this.specVersion = specVersion.asText();
+            } else {
+                this.specVersion = "1.0";
```

Comment:
Why change the version?  Should just leave the version as it is.  IMO, We only do these version checks when a spec version ends up not being implemented by us and is not backward compatible

Reply:
Just leave it as a string as its sent across the wire.

Reply:
OK

Reply:
@patriot1burke fixed